src/OutSystems/GridAPI/Selection.ts

```diff
@@ -132,12 +132,12 @@ namespace OutSystems.GridAPI.Selection {
             errorCode:
                 OSFramework.DataGrid.Enum.ErrorCodes
                     .API_FailedGetSelectionAverage,
-            hasValue: true,
             callback: () => {
                 const grid = GridManager.GetGridById(gridID);
 
                 return grid.features.selection.getSelectionAverage();
-            }
+            },
+            hasValue: true
```

Comment:
Same here

Reply:
This one, I'm not sure, as -1 is a valid value... better would be not to return any value at all... but not possible... as such left 0 as an error value - discuss-able.